src/user/model.coffee

```diff
@@ -11,9 +11,12 @@ BLANK_USER =
   name: null
   profile_url: null
   courses: []
+  _course_data: []
+  endpoints: {}
+  isLoaded: false
```

Comment:
I don't think we do on logout?  But even if we do, `endpoints` would be blank for a period of time while the call is pending.  If someone was fast (or the network was slow) they could attempt to log back in before the `status/auth` call completed.